modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -1928,13 +1928,36 @@ private static boolean areCompatible(Object[] objs, Class[] types) {
         for (int i = 0, size = objs.length; i < size; i++) {
             Object o = objs[i];
 
-            if (o != null && !types[i].isInstance(o))
+            if (o != null && !box(types[i]).isInstance(o))
                 return false;
         }
 
         return true;
     }
 
+    private static Class<?> box(Class cls) {
```

Comment:
Maybe we should replace it with `org.apache.ignite.internal.util.IgniteUtils#box`?

Reply:
Thank you, I completely forgot to search for existing methods!